addons/website_slides_survey/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -1,15 +1,61 @@
 odoo.define('website_slides_survey.fullscreen', function (require) {
     "use strict";
-    
+
     var core = require('web.core');
-    var _t = core._t;
+    var QWeb = core.qweb;
     var Fullscreen = require('website_slides.fullscreen');
-    
+
     Fullscreen.include({
         xmlDependencies: (Fullscreen.prototype.xmlDependencies || []).concat(
             ["/website_slides_survey/static/src/xml/website_slides_fullscreen.xml"]
         ),
+        /**
+         * Fetches the certification url in case it wasn't present in the data
+         *
+         * @private
+         */
+        _fetchCertificationUrl: function (){
+            var slide = this.get("slide");
+            return this._rpc({
+                route: "/slides_survey/certification_url/get",
+                params: {
+                    slide_id: slide.id
+                }
+            }).then(function (data){
+                if (!data.error){
+                    slide.certificationUrl = data[0].certification_url;
+                    slide.certificationDone = data[0].certification_done;
+                    slide.certificationId = data[0].survey_id;
+                }
+            });
+        },
+        /**
+         * @override
+         */
+        _fetchSlideContent: function (){
+            this._super.apply(this, arguments);
```

Comment:
the `super()` should be called at the end of the method. Otherwise, if the slide type is not certif, the returned value will be $.when, which break webpage type.
So your method check if certif. If so, do and return certif business, else deletage the return to super()